src/registry/BaseRegistry.ts

```diff
@@ -28,7 +28,7 @@ export abstract class BaseRegistry implements IRegistry {
     return 'chains';
   }
 
-  protected getWarpArtifactsPaths({ tokens }: WarpCoreConfig) {
+  getWarpArtifactsPaths({ tokens }: WarpCoreConfig) {
```

Comment:
What's motivating this change? If we want something public, we should make all the *Path methods public